src/platform/backends/shared/base_snapshot.cpp

```diff
@@ -134,6 +145,7 @@ QJsonObject multipass::BaseSnapshot::serialize() const
 
     snapshot.insert("name", QString::fromStdString(name));
     snapshot.insert("comment", QString::fromStdString(comment));
+    snapshot.insert("creation_timestamp", creation_timestamp.toString(Qt::ISODate));
```

Comment:
I just realized this was dropping milliseconds. There is a `Qt::ISODateWithMs` which is closer to what you had initially.

Reply:
BTW, I believe this explains the wrong order after loading snapshots that were taken in quick succession:

```sh
$ mp snapshot b -n foo && mp snapshot a -n bar && mp snapshot b -n bar && mp snapshot b -n baz && mp snapshot a -n baz && mp snapshot a -n foo
Snapshot taken: b.foo
Snapshot taken: a.bar
Snapshot taken: b.bar
Snapshot taken: b.baz
Snapshot taken: a.baz
Snapshot taken: a.foo
$ multipass info --snapshot-overview --all                                                 Instance   Snapshot   Parent   Comment
a          bar        --       --
a          baz        bar      --
a          foo        baz      --
b          foo        --       --
b          bar        foo      --
b          baz        bar      --
$ snap stop multipass.multipassd
2023-06-28T12:43:29+01:00 INFO Waiting for "snap.multipass.multipassd.service" to stop.
Stopped.
$ snap start multipass.multipassd
Started.
$ multipass info --snapshot-overview --all
Instance   Snapshot   Parent   Comment
a          bar        --       --
a          foo        baz      --
a          baz        bar      --
b          foo        --       --
b          baz        bar      --
b          bar        foo      --
```